testsuite/shared/pom.xml

```diff
@@ -259,6 +259,10 @@
             <groupId>xom</groupId>
             <artifactId>xom</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wildfly.security</groupId>
+            <artifactId>wildfly-elytron</artifactId>
+        </dependency>
```

Comment:
Once the new servlet is moved as mentioned in a previous comment, I think this won't be needed.